recipe/meta.yaml

```diff
@@ -16,27 +16,32 @@ build:
   run_exports:
     # forward compatible: https://abi-laboratory.pro/tracker/timeline/libxslt/
     - {{ pin_subpackage('libxslt') }}
+  ignore_run_exports:
+    - icu  # [osx]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - make        # [not win]
     - m2-patch    # [win]
     - patch       # [not win]
-    - libtool     # [not win]
+    - gnuconfig   # [not win]
   host:
     - icu         # [not win]
```

Comment:
AFAIK, libxslt does not have a direct dependency on icu. It's only dependencies are libxml2 and optionally libgcrypt. What happens, if we remove icu here?

Reply:
Without icu an error will be:
`["  ERROR (libxslt,lib/libxslt.so.1.1.35): .. but ['icu'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)", "  ERROR (libxslt,lib/libxslt.so.1.1.35): .. but ['icu'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)", "  ERROR (libxslt,lib/libxslt.so.1.1.35): .. but ['icu'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)", "  ERROR (libxslt,lib/libexslt.so.0.8.20): .. but ['icu'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)", "  ERROR (libxslt,lib/libexslt.so.0.8.20): .. but ['icu'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)", "  ERROR (libxslt,lib/libexslt.so.0.8.20): .. but ['icu'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)"]`

Reply:
Ok, that is very annyoing, as it is probably overlinking. Then I guess we should keep it and ignore it, as you did.